### PR TITLE
Allow TSV download for scatter plot

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: app/scripts/views/components/scatterPlot/scatterPlot.js [hunk 1]
```diff
@@ -165,9 +185,39 @@
...
+      } else if (fileType === 'tsv') {
+        initTsvDownloadData();
       }
     };
 
+    function initTsvDownloadData() {
+      var _title = opts_.title || 'Mutation Count vs. CNA';
+      var _data = ['Patient ID', 'Sample ID', 'Mutation Count', 'CNA'];
+
+      if (groups_.length > 1) {
```
Comment (kalletlak) [inline app/scripts/views/components/scatterPlot/scatterPlot.js]: Instead of `Group` why not we specify it as `Selected` and values as `0/1 or true/false`

Comment (kalletlak) [inline app/scripts/views/components/scatterPlot/scatterPlot.js]: and why not add it by default, I mean for any number of groups?

**Review by kalletlak — CHANGES_REQUESTED** [review]

File: app/scripts/views/components/scatterPlot/scatterPlot.js (hunk 1 above)
Comment (zhx828) [reply app/scripts/views/components/scatterPlot/scatterPlot.js]: I struggled a little bit. If the user just load the page and download the file, should we treatment as all selected?

Comment (zhx828) [reply app/scripts/views/components/scatterPlot/scatterPlot.js]: I think about it. I was going to add download for survival plot, and survival has group concept for rainbow survival. Just for consistency, I changed to group in here as well. What do you think?

Comment (kalletlak) [reply app/scripts/views/components/scatterPlot/scatterPlot.js]: I think yes!

Comment (zhx828) [reply app/scripts/views/components/scatterPlot/scatterPlot.js]: OK, that would make life much easier. lol. I will change that

Comment (kalletlak) [reply app/scripts/views/components/scatterPlot/scatterPlot.js]: Ok.. that make sense

**Review by kalletlak — APPROVED** [review]

**Pull request merged**